ddtrace/tracer/stats.go

```diff
@@ -278,13 +279,22 @@ func (sb *rawBucket) Export() statsBucket {
 	return csb
 }
 
+type trilean int32
+
+const (
+	trilean_not_set trilean = iota
+	trilean_true
+	trilean_false
+)
+
 type rawGroupedStats struct {
 	hits            uint64
 	topLevelHits    uint64
 	errors          uint64
 	duration        uint64
 	okDistribution  *ddsketch.DDSketch
 	errDistribution *ddsketch.DDSketch
+	IsTraceRoot     trilean
```

Comment:
@knusbaum I didn't expect ternary logic 😆 What's the reason behind it? I assume it's avoiding possible nil pointers. Also, why an `int32`? Wouldn't an `int8` be enough?

Reply:
Ok, cool! 😁 Good to know.